app_api_template.coffee

```diff
@@ -46,16 +54,15 @@ startApp = ->
 		console.error 'Caught api exception: ' + err.stack
 		return
 
+	#TO_BE_REPLACED_BY_PREPAREMODULEINCLUDES
 
-	httpServer = http.createServer(app).listen(app.get('port'), ->
+	http.createServer(app).listen(app.get('port'), ->
 		console.log("ACAS API server listening on port " + app.get('port'))
 		bootstrap = require "./src/javascripts/ServerAPI/Bootstrap.js"
 		if bootstrap.main?
 			bootstrap.main()
 	)
 
```

Comment:
A number of changes to migrate from express 3 to expres 4

https://expressjs.com/en/guide/migrating-4.html